packages/cactus-test-tooling/README.md

```diff
@@ -7,3 +7,15 @@
 ```
 // TODO: DEMONSTRATE API
 ```
+## Docker image for the ws-identity server
+
+A docker image of the [ws-identity server](https://hub.docker.com/repository/docker/brioux/ws-identity) is used to test integration of WS-X.509 credential type in the fabric connector plugin.
+
```

Comment:
@petermetz I have included basic instructions on how build the ws-identity docker image here